src/named-groups/interfaces/INamedGroupsClient.ts

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { AccessToken } from "@itwin/core-bentley";
+import { NamedGroup, NamedGroupCreate, NamedGroupList, NamedGroupMinimal, NamedGroupMinimalList, NamedGroupUpdate } from "./NamedGroups";
+import { EntityListIterator } from "../../common/iterators/EntityListIterator";
+import { PreferReturn } from "../../common/Common";
+
+export interface INamedGroupsClient {
+  /**
+    * Creates a NamedGroup for an iTwin.
+    * @param {string} accessToken OAuth access token with imodels:modify or itwin-platform scope.
```

Comment:
API does not work with just the `imodels:modify` scope. Only `itwin-platform` scope is required.

Reply:
```suggestion
    * @param {AccessToken} accessToken OAuth access token with scope `itwin-platform`.
```

Reply:
Fixed all the instances of this.